src/lamp_py/aws/s3.py

```diff
@@ -476,7 +476,10 @@ def write_parquet_file(
     partition_strings = []
     for col in partition_cols:
         unique_list = pc.unique(table.column(col)).to_pylist()
-        assert len(unique_list) == 1
+        if len(table) > 0:
+            assert (
+                len(unique_list) == 1
+            ), f"Table {s3_dir} column {col} had {len(unique_list)} unique elements"
```

Comment:
Won't this result in us writing a table to a location we don't want? Since the `partition_strings` is built as part of the final file path.

If we're having an issue with writing empty GTFS files I think we should abort this process further up the chain, and not write an empty file at all.